src/pages/iou/request/step/IOURequestStepConfirmation.js

```diff
@@ -107,6 +107,11 @@ function IOURequestStepConfirmation({
         // If there is not a report attached to the IOU with a reportID, then the participants were manually selected and the user needs taken
         // back to the participants step
         if (!transaction.participantsAutoAssigned) {
+            // When going back to the participants step, if the iou is a "request" (not a split), then the participants need to be cleared from the
+            // transaction so that the participant can be selected again.
+            if (iouType === CONST.IOU.TYPE.REQUEST) {
+                IOU.setMoneyRequestParticipants_temporaryForRefactor(transactionID, []);
+            }
```

Comment:
doing expedite review from [request](https://expensify.slack.com/archives/C02NK2DQWUX/p1702424977654789)

Participants briefly show and are cleared after a moment (navigation is finished).
This doesn't happen on production. (cleared immediately, no glitch)
Btw, I don't think we should block the deploy because of this so going ahead

https://github.com/Expensify/App/assets/108292595/bef5de5e-1f1b-4c16-bb30-91004cf76068

Reply:
I was actually going to say the same thing.
Let me investigate if there's any way to fix this without adding any new variable to transaction and will update it here

Reply:
We could raise this as a separate issue if it is still occuring. Let me know!

Reply:
I noticed that as well. I think it would be nice to open a new bug report for that and see if contributors can find a good solution to that. Would you please report the bug so we can get started on that?